### PR TITLE
Added code snippet to update build_info.json with maintainer field and Updating  build_info.json files

### PR DESCRIPTION
**Pull request by shubham-garud**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by shubham-garud** [comment]
Please review @nikhil-kalbande @Ryan-Dmello .

File: script/generate_build_info.py [hunk 1]
```diff
@@ -131,9 +166,22 @@ def get_files_list(dirname:str, recursive:bool=True):
         if 'version' not in docker_details:
             docker_details ['version'] = '*'
         dockerfile_versions.append(docker_details)
-
-
+    elif file.endswith(".json"):
+
+        new_key_value = {"maintainer":"{}".format(get_maintainer_for_package(dir_name))}
```
Comment (Ryan-Dmello) [inline script/generate_build_info.py:171]: As the maintainer can be either from dockerfile or build-script, this variable should be based on the conditional value setting. Which is defined between line 179 - 189.

i.e change the line to `new_key_value = {"maintainer": maintainer}`

**Review by Ryan-Dmello — APPROVED** [review]
Looks ok to me

**Comment by nikhil-kalbande** [comment]
@shubham-garud  Currently we have 192 packages in dashboard, so we will need to update same number of build_info files. Can you please rerun automation to get those update in this PR only.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**